django/thunderstore/repository/package_manifest.py

```diff
@@ -39,15 +42,9 @@ def __init__(self, *args, **kwargs):
 
     def validate(self, data):
         result = super().validate(data)
-        if not self.uploader.can_user_upload(self.user):
-            raise ValidationError(
-                f"Missing privileges to upload under author {self.uploader.name}"
-            )
         reference = PackageReference(
-            self.uploader.name, result["name"], result["version_number"]
+            result["author_name"], result["name"], result["version_number"]
         )
-        if reference.exists:
```

Comment:
Why has this check been removed?

Reply:
Ah, looks like it was just moved to the form

Reply:
It's now part of the upload form ([source](https://github.com/risk-of-thunder/Thunderstore/pull/144/files/8f5c149f2c81d92a53cfad085fe7ba878b598291#diff-b2cd1a3295a5097c8863739b263ed83b89471274f0fa5805692cf509e9486481R86-R89))